### ISSUE TITLE
Change release to push all bits into Maven

### ISSUE THREAD

**Issue by enebo** [jruby/jruby#4829]
@mkristian started a proof of concept of adding windows installer as an artifact: #4790.  We also need all existing bit to have extra fingerprints.

As a related part of this we also need to submit PRs to the ruby managers to point at the maven urls where the bits will be at.

**Comment by enebo** [comment]
This is partially done in that we pushed fingerprints into our maven artifacts but we have not tried to get any ruby managers to start using the new url scheme yet so I am punting that part down to next point release.  It should be retroactive back to 9.1.14.0 once finished.

**Comment by headius** [comment]
What more is outstanding for this? The Ruby managers have not addressed the related issues, but we have filed them and there's movement (in rvm at least). We are also in communication with Engine Yard about keeping their sponsored S3 around for a bit longer.

**Comment by enebo** [comment]
Yeah I guess finish windows installer artifacts and once those PRs happen on ruby managers we should be good.

**Comment by headius** [comment]
Still waiting on Windows installer logic in maven push and for Ruby installers to catch up with new URLs.

**Comment by headius** [comment]
@enebo @mkristian Ping?

**Comment by headius** [comment]
@mkristian Can we coax you into helping us resolve this long-standing issue? We are still dependent on a third party to host the Windows installer, and if they ever drop us we'll have no good way to publish it.

I feel like it's nearly done... help us Obi-Wan of Maven!

**Comment by mkristian** [comment]
I do remember there was some windows tool to execute to produce the jruby.exe

The linked PR  #4790 is just adds a new profile to upload the jruby.exe

I guess I am not quite sure what's the missing piece.
